sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -440,6 +442,76 @@ public BoundedWindow window() {
   }
 
   /**
+   * Annotation for declaring and dereferencing timers.
+   *
+   * <p><i>Not currently supported by any runner.</i>
+   *
+   * <p>To declare a timer, create a field of type {@link TimerSpec} annotated with a {@link
+   * TimerId}. To use the cell during processing, add a parameter of the appropriate {@link Timer}
+   * subclass to your {@link ProcessElement @ProcessElement} method, and annotate it with {@link
```

Comment:
This comment seems still out of sync with the DoFnSignatures code - DoFnSignatures doesn't look for timers in parameters of `ProcessElement`, only in fields.